frontends/election-manager/src/app_root.tsx

```diff
@@ -44,8 +44,11 @@ import {
 } from './config/types';
 import { useElectionManagerStore } from './hooks/use_election_manager_store';
 import { getExportableTallies } from './utils/exportable_tallies';
+import { ElectionManagerStoreBackend } from './lib/backends/types';
 
 export interface Props {
+  logger: Logger;
+  backend: ElectionManagerStoreBackend;
   storage: Storage;
```

Comment:
I think you can remove `storage` now that it's not used

Reply:
Done.